build.gradle

```diff
@@ -752,7 +755,13 @@ subprojects {
 
             dependency "com.alibaba:druid-spring-boot-starter:1.2.23"
             dependency "org.springframework.boot:spring-boot-starter-jetty:2.7.18"
+
+            dependency "com.google.protobuf:protobuf-java-util:${protobufVersion}"
+            dependency "com.google.protobuf:protobuf-java:${protobufVersion}"
+            dependency "com.google.protobuf:protoc:${protocVersion}"
+            
             dependency "com.mysql:mysql-connector-j:8.4.0"
```

Comment:
The google.protobuf dependencies should be placed after the grpc dependencies. The same as the `protobufVersion` var.